chapter2/notes.md

```diff
@@ -17,18 +45,55 @@
 
 ## Frameworks
 
```

Comment:
@dami-gg Jquery, lodash/underscore, React are libraries not frameworks, I think it's important to distinguish.